YouTag/YYTFilterTagView.swift

```diff
@@ -10,8 +10,8 @@ import UIKit
 
 class YYTFilterTagView: YYTTagView {
 	
-	override init(frame: CGRect, tagsList: NSMutableArray, isAddEnabled: Bool, isMultiSelection: Bool, isDeleteEnabled: Bool) {
-		super.init(frame: frame, tagsList: tagsList, isAddEnabled: isAddEnabled, isMultiSelection: isMultiSelection, isDeleteEnabled: isDeleteEnabled)
+    override init(frame: CGRect, tagsList: NSMutableArray, isAddEnabled: Bool, isMultiSelection: Bool, isDeleteEnabled: Bool, areSuggestionsEnabled: Bool, isForArtists: Bool) {
+        super.init(frame: frame, tagsList: tagsList, isAddEnabled: isAddEnabled, isMultiSelection: isMultiSelection, isDeleteEnabled: isDeleteEnabled, areSuggestionsEnabled: areSuggestionsEnabled, isForArtists: isForArtists)
```

Comment:
Should be like this:
```
    override init(frame: CGRect, tagsList: NSMutableArray, isAddEnabled: Bool, isMultiSelection: Bool, isDeleteEnabled: Bool) {
         super.init(frame: frame, tagsList: tagsList, isAddEnabled: isAddEnabled, isMultiSelection: isMultiSelection, isDeleteEnabled: isDeleteEnabled, areSuggestionsEnabled: false, suggestionArray: nil)

```

Reply:
I just thought about an improvement to this init. I'll do it now and pull it so you can use that